pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2019 eXo Platform SAS.
+
+    This is free software; you can redistribute it and/or modify it
+    under the terms of the GNU Lesser General Public License as
+    published by the Free Software Foundation; either version 2.1 of
+    the License, or (at your option) any later version.
+
+    This software is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this software; if not, write to the Free
+    Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+    02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>addons-parent-pom</artifactId>
+        <groupId>org.exoplatform.addons</groupId>
+        <version>10</version>
+    </parent>
+    <groupId>org.exoplatform.addons.lead-capture</groupId>
+    <artifactId>lead-capture</artifactId>
+    <version>1.0.x-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>eXo Lead Capture Addon</name>
+    <description>eXo Addon for Lead Capture and management</description>
+    <modules>
+        <module>war</module>
+        <module>services</module>
+        <module>packaging</module>
+    </modules>
+    <scm>
+        <connection>scm:git:git://github.com/exo-addons/lead-capture.git</connection>
+        <developerConnection>scm:git:git@github.com:exo-addons/lead-capture.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/exo-addons/lead-capture</url>
+    </scm>
+    <properties>
+        <platform.version>6.0.0-M09</platform.version>
+        <org.lombok.version>1.18.2</org.lombok.version>
+        <org.jackson.version>2.4.2</org.jackson.version>
+        <addon.exo.tasks.version>3.0.x-SNAPSHOT</addon.exo.tasks.version>
```

Comment:
It should be coherent with the plaform.version property

Reply:
Ok